### ISSUE TITLE
Parsing error: Using the export keyword between a decorator and a class is not allowed

### ISSUE THREAD

**Issue by pairmix** [nuxt-community/nuxt-property-decorator#65]
Hi there,
I get this issue:
Parsing error: Using the export keyword between a decorator and a class is not allowed

````
<script lang="ts">
  import { Component, Prop, Vue } from 'nuxt-property-decorator';

  @Component
  export default class Contact extends Vue {
    ...
</script>````

**Comment by domi91c** [comment]
@pairmix Any luck solving this issue?

**Comment by husayt** [comment]
I just released v 2.7.1 of the library which fixes many issues. If your issues still exists, please, provide a link to codesandbox with the issue

**Comment by dirodriguezm** [comment]
I fixed it by adding 

```js
parserOptions: {
    ecmaFeatures: { legacyDecorators: true },
}
```
